src/atomate2/common/schemas/pheasy.py

```diff
@@ -349,8 +349,10 @@ def from_forces_born(
             int(supercell_matrix[1][1]),
             int(supercell_matrix[2][2]))
         
-        # Generate the sensing matrix for the input of machine leaning method.i.e., LASSO,
-        pheasy_cmd_3 = 'pheasy --dim "{0}" "{1}" "{2}" -w 2 -d --symprec 1e-3 --ndata "{3}" --disp_file'.format(
+        # Generate the Compressive Sensing matrix,i.e., displacment matrix
+        # for the input of machine leaning method.i.e., LASSO,
+        pheasy_cmd_3 = 'pheasy --dim "{0}" "{1}" "{2}" -w 2 -d --symprec 1e-3 \
```

Comment:
One should be able to change symprec here as well

Reply:
Resolve.

Reply:
Done it.